src/main/resources/anniv6Resources/localization/eng/BrokenSpace/Relicstrings.json

```diff
@@ -0,0 +1,105 @@
+{
+  "${ModID}:BrokenDataDisk": {
+    "NAME": "Broken Data Disk",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the start of each combat, lose #b",
+      " #yFocus. The first time you channel an orb each turn, channel another orb of the same type."
+    ]
+  },
+  "${ModID}:BrokenSneckoSkull": {
+    "NAME": "Broken Snecko Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you gain a buff, gain #b",
+      " more."
+    ]
+  },
+  "${ModID}:BrokenRedSkull": {
+    "NAME": "Broken Red Skull",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you are above #b",
+      "% #yHP, you have #b",
+      " extra #yDexterity."
+    ]
+  },
+  "${ModID}:BrokenOmamori": {
+    "NAME": "Broken Omamori",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "Add #b",
+      " #yCurses to your deck. Whenever you rest, upgrade #b",
+      " random cards in your deck."
+    ]
+  },
+  "${ModID}:BrokenDamaru": {
+    "NAME": "Broken Damaru",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the end of your turn, you may #yExhaust a card in your hand. Gain #yMantra equal to it's cost.",
+      "Select a card to exhaust for mantra."
+
+    ]
+  },
+  "${ModID}:BrokenWingBoots": {
+    "NAME": "Broken Wing Boots",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "When you would enter an event in a ? room, choose between two events."
+
+    ]
+  },
+  "${ModID}:BrokenSozu": {
+    "NAME": "Broken Sozu",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "In-Combat Potions cost #b",
+      " [E] to use. When you spend energy on a potion, use another random potion.",
+      " Energy"
+    ]
+  },
+  "${ModID}:BrokenSneckoEye": {
+    "NAME": "Broken Snecko Eye",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the start of each turn, add #b",
+      " copy of a random card from your master deck into your hand. It #yExhausts, is #yEthereal, and is unreadable."
```

Comment:
This can just say "deck" ("master deck" is something modders tend to say because that's that's the field you access in the code, `AbstractPlayer.masterDeck`).